packages/web/index.js

```diff
@@ -100,6 +113,14 @@ module.exports = (neutrino, opts = {}) => {
       .use(EnvironmentPlugin, [options.env]);
   }
 
+  if (options.devtool) {
```

Comment:
I don't think this conditional is needed, since the handling above will mean that `devtool: false` will be converted to an object anyway, and thus `options.devtool` will always be truthy (other than for spurious values like `0`, `""`, `null` etc, which we shouldn't worry about handling anyway.